src/GeneralStateTestsFiller/stReturnDataTest/modexp_modsize0_returndatasizeFiller.json

```diff
@@ -0,0 +1,83 @@
+{
+    "modexp_modsize0_returndatasize" : {
+        "env" : {
+            "currentCoinbase" : "2adc25665018aa1fe0e6bc666dac8fc2697ff9ba",
+            "currentDifficulty" : "0x20000",
+            "currentGasLimit" : "10000000000",
+            "currentNumber" : "1",
+            "currentTimestamp" : "1000",
+            "previousHash" : "5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6"
+        },
+        "expect" : [
+	    {
+                "indexes" : {
+                    "data" : -1,
+                    "gas" : 0,
+                    "value" : -1
+                },
+                "network" : [ "EIP150", "EIP158"],
+                "result" : {
+                    "1000000000000000000000000000000000000000" : {
+                        "storage" : {
+			    "0x01" : "0x00",
+			    "0x02" : "0x00",
+                            "0x03" : "0xffffffff"
+                        }
+                    }
+                }
+            },
+	    {
+                "indexes" : {
+                    "data" : -1,
+                    "gas" : 0,
+                    "value" : -1
+                },
+                "network" : ["Byzantium"],
+                "result" : {
+                    "1000000000000000000000000000000000000000" : {
+                        "storage" : {
+			    "0x01" : "0x01",
+                            "0x02" : "0x00",
+			    "0x03" : "0x00"
+                        }
+                    }
+                }
+            }
+        ],
+        "pre" : {
+	    "1000000000000000000000000000000000000000" : {
+                "balance" : "",
+                "code" : "{ (CALLDATACOPY 0 0 (CALLDATASIZE)) [[1]] (CALLCODE (GAS) 5 0 0 (CALLDATASIZE) 1000 32) [[2]](MLOAD 1000) [[3]](RETURNDATASIZE) }",
+                "nonce" : "0",
+                "storage" : {
+		  "0x03" : "0xffffffff"
+                }
+            },
+            "a94f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
+                "balance" : "1000000000000000000000",
+                "code" : "",
+                "nonce" : "0",
+                "storage" : {
+                }
+            }
+        },
+        "transaction" : {
+	    "// data" : [
+		"modlength zero"
+	    ],
+            "data" : [
+"0x0000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000000101"
+            ],
+            "gasLimit" : [
+                "100000000", "90000", "110000", "200000"
```

Comment:
do we really need that many gas checks? each item is another fork * tests

Reply:
Maybe not really.